### PR TITLE
DEV-84-RedesignHomePage

### PR DESCRIPTION
**Pull request by mxabi**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Screenshots
__Ad screenshots of the changes if relevant__

**Before**

**After**

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Dependency Changes
* Please list any dependency changes and why they were required

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

### CODE REVIEW

File: .vs/VSWorkspaceState.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (InManuBytes) [inline .vs/VSWorkspaceState.json:1]: I think these files might be particular to your local comp? maybe you can add the .vs to the gitignore file?

Comment (InManuBytes) [reply .vs/VSWorkspaceState.json:1]: I think the same goes for some other files in here, we can look at that later.

File: src/routes/Home/Home.jsx [hunk 2]
```diff
@@ -1,114 +1,228 @@
...
-  },
-  break: {
-    [theme.breakpoints.up('xs')]: {
-      width: '0px',
-    },
-    [theme.breakpoints.up('mobile')]: {
-      width: '25px',
-    },
-  },
+const Img = styled('img')({
```
Comment (InManuBytes) [inline src/routes/Home/Home.jsx:23]: Does this work "responsively"?

**Review by InManuBytes — COMMENTED** [review]
The code looks fine, but since these are frontend changes you'd want to add some screenshots of the changes, maybe even a gif/video of how it reacts to different width of screens.

File: .vs/VSWorkspaceState.json (hunk 1 above)
Comment (abbeyalam) [reply .vs/VSWorkspaceState.json:1]: Got it!

**Comment by InManuBytes** [comment]
Closed since we have https://github.com/outintech/nbjc-webapp/pull/74

**Pull request closed without merging**